<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gsDockWidgetGeometryEditSettings</class>
 <widget class="QgsDockWidget" name="gsDockWidgetGeometryEditSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaavakohteiden jakamisen muutosasetukset</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxKeepFeatureRelations">
        <property name="text">
         <string>Säilytä kaavamääräys ja teemat kaavakohdetta jakaessa</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPreventNULLGeometries">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Estä NULL-geometriat kaavakohdetta jakaessa</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Todella pienet kohteet jakamisen seurauksena</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBoxPreventFeaturesWithTinyGeometries">
           <property name="text">
            <string>Estä äärimmäisen pienet kohteet</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Suurin sallittu % kokoero alkuperäiseen kohteeseen</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBoxPreventFeaturesWithTinyGeometries">
             <property name="value">
              <number>99</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
